capa/features/common.py

```diff
@@ -92,23 +97,32 @@ def __nonzero__(self):
 
 
 class Feature(abc.ABC):
-    def __init__(self, value: Union[str, int, float, bytes], description=None):
+    def __init__(
+        self, value: Union[str, int, float, bytes], access: Optional[str] = None, description: Optional[str] = None
+    ):
         """
         Args:
           value (any): the value of the feature, such as the number or string.
           description (str): a human-readable description that explains the feature value.
```

Comment:
```suggestion
          access (str): set if feature is read or written
          description (str): a human-readable description that explains the feature value.
```

Reply:
suggest that we make `access` an enum to capture its possible values